windows-icon-handler/check-icon-handler/check-icon-handler.rc

```diff
@@ -0,0 +1,143 @@
+// Microsoft Visual C++ generated resource script.

```

Comment:
Missing header

Reply:
This one is a bit annoying :'( Visual studio overwrite our header each time I rebuild the solution.

Reply:
`pre-commit` hook ?

Reply:
Does the other `rc` files are auto generated ?

Reply:
Only the boilerplate code is generated. This file need to be kept it give the product name, company name etc ... I've added licence header